client/scss/generic/_normalize.scss

```diff
@@ -1,279 +1,110 @@
-/* stylelint-disable declaration-block-no-duplicate-properties, property-no-vendor-prefix, scale-unlimited/declaration-strict-value */
```

Comment:
This isn’t really possible to remove as our CSS linting picks up a lot of issues in this file otherwise.

Reply:
Question: Should we be pulling this from `npm` instead of vendoring it?

Reply:
I assume pulling from npm would make it harder to review backwards-incompatible changes as highlighted in this PR, as well as the potential of removing it entirely.